.editorconfig

```diff
@@ -8,6 +8,7 @@ charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
+quote_type = single
```

Comment:
I had no idea this was an option. I've been copy/pasting the same config file for so long :laughing: